### DISCUSSION TITLE
guide/api/client/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#227] [docs-comment]
# guide/api/client/

使用 布局组件 <Layout />： 页面布局组件 <NotFound /> ： 404 页面布局组件 通用组件 <VPLink /> ： 链接组件 <VPButton />: 按钮组件 <VPIcon />: 图标组件 <VPBadge />: 徽标组件 <VPImage />: 图片组件 <VPHomeBox />: 首页布局组件 更多其他组件请...

https://theme-plume.vuejs.press/guide/api/client/

**Comment by MrHaoH** [comment]
大佬，请教个问题，我如何一进页面默认就是暗黑模式的主题？在哪里设定isDark?感谢大佬的回复

**Reply by MrHaoH** [reply]
我找到了，不用了大佬，在这配置https://theme-plume.vuejs.press/config/basic/#appearance

**Comment by MrHaoH** [comment]
大佬，请教个问题，在自定义组件中如何获取.vuepress/config.ts文件中的base字段？
export default defineUserConfig({
  base: "/aaa/"
})
我想获取“/aaa/”，感谢大佬

**Reply by pengzhanbo** [reply]
```ts
import { withBase } from 'vuepress/client'

const path = withBase('/a.html') // -> /aaa/a.html
```
